src/allotropy/allotrope/schema_mappers/adm/plate_reader/benchling/_2023/_09/plate_reader.py

```diff
@@ -64,10 +65,10 @@ class MeasurementType(Enum):
 
 
 MeasurementDocumentItems = (
-    UltravioletAbsorbancePointDetectionMeasurementDocumentItems
-    | FluorescencePointDetectionMeasurementDocumentItems
-    | LuminescencePointDetectionMeasurementDocumentItems
-    | OpticalImagingMeasurementDocumentItems
+        UltravioletAbsorbancePointDetectionMeasurementDocumentItems
```

Comment:
You seem to have some kind of odd setting that's causing you to add an extra indent when formating. Please revert these changes. 

It might be easiest to find the setting and update it, and then run hatch run fix

Unfortunately I've never seen this kind of change, so I'm not sure what the setting is.